src/commands/paranet/paranet-sync-command.js

```diff
@@ -158,7 +158,6 @@ class ParanetSyncCommand extends Command {
     }
 
     async syncAssetState(
-        operationId,
```

Comment:
You removed operationId but it's stil used in the code, I see it's used still in logs (line 311)